### ISSUE TITLE
Allow coverage list to be filtered by service

### ISSUE THREAD

**Issue by patcon** [archivers-space/coverage#9]
This came to mind while reading this [Science to Action (S2A) content brief](https://docs.google.com/document/d/10-RwzjecOEWOozJuwuNfzQ7nNZDfMX_QRflxfrPtKXc/edit).

The thinking is that if we want `coverage` to be a place that's useful to other archiving project maintainers, then we should probably have a way to filter by "service" (in coverage parlance),  so that these other communities can audit their impact in relation to other projects.

This would allow stakeholders and maintainers of other systems to browse the coverage interface and vet with confidence that it speaks accurately to the progress of their project. Their ability to do this could allow other stakeholders (like S2A) to have more faith in progress, because everyone says "Yes, this seems to accurate reflect our work" :)

**Comment by b5** [comment]
Agreed! I'm in the process of shipping a bunch of upgrades, some of which starts down this path, but I hadn't thought about self-audit angle. Great point. These details will be surfaced over on the [api](https://github.com/archivers-space/api), but the implementation of coverage will remain here.

I'm very excited to start shipping more ways to slice & dice the coverage repo. The first is a way to calculate coverage based on a url pattern (e.g. "epa.gov"), which we need for broader coverage calculation purposes, but I'll definitely make isolating coverage to one or more data repositories the next feature we ship for coverage. From there we can move on to improving the repository interface & repository implementations themselves. (With some sort of redis-based caching instead of these silly static files!)
